### DISCUSSION TITLE
[CLIENT-MOD] - Characters Screen Modifications

### DISCUSSION THREAD

**Discussion by ghost** [azerothcore/azerothcore-wotlk#3940] [Tools]
Heyio,

I share with you a rather nice creation.

I redid the character creation interface and I got closer to what Shadowlands proposes for its beta.
A modification of the backgrounds of creations and selections of the characters was also made.

I would ask you to leave the copyright in the code.
It took me 12 hours of work for all logo and modifications.

I did this for the Vishanka server (a French server), I have their agreement to share my work with you :)

[I WANT TO DOWNLOADING THIS AWESOME INTERFACE ! <3](https://github.com/iThorgrim-Hub/Wrath-of-the-Lich-King-Reforged)

![unknown](https://user-images.githubusercontent.com/58987080/82546826-0c5c6d80-9b59-11ea-85d2-a039bf90fc8d.png)

![unknown (1)](https://user-images.githubusercontent.com/58987080/82547442-05822a80-9b5a-11ea-98d3-7f0bf3e54ca3.png)

![image](https://user-images.githubusercontent.com/58987080/82547516-2c406100-9b5a-11ea-9fda-13f312619172.png)

![image](https://user-images.githubusercontent.com/58987080/82547528-319dab80-9b5a-11ea-8c31-4cc08d0ec1b7.png)

![image](https://user-images.githubusercontent.com/58987080/82547499-25195300-9b5a-11ea-9557-7b2a7b5ed4c6.png)

**Comment by BarbzYHOOL** [comment]
Wow @iThorgrim-Hub that's beautiful, do we need the WOD races or not?

@Helias check this out :D @FrancescoBorzi

**Comment by ghost** [comment]
No it's not mandatory but it's still more beautiful with :)

**Comment by imxari** [comment]
Would be cool to have mounts imported from Cata and MoP. Was looking into how to do it but, there's not really that much detail on it.

**Comment by chamorrorenzo** [comment]
so cool!
What version does this support? 3.3.5 right?
cause im seeing a worgen there

**Comment by ghost** [comment]
@chamorrorenzo Yes 3.3.5 only :) Worgen is for Vishanka (custom wotlk server)

**Comment by FrancescoBorzi** [comment]
good job @iThorgrim-Hub !

**Comment by ghost** [comment]
Thanks @FrancescoBorzi :D

**Comment by Helias** [comment]
little bug
![image](https://user-images.githubusercontent.com/519778/82947297-18548f00-9fa0-11ea-8053-9ddc0a7dd0ab.png)

anyway you could leave the normal logo instead of Vispanka :sweat_smile:

**Comment by ghost** [comment]
I'll do an update to fix this problem 👍

**Comment by ghost** [comment]
New version available!

- Fix some "Responsive" Bug
- Now class frame is show when you customize your characters.
- Now when you right click on customization goes to previous customization.

_If anyone knows how to add a "zoom" function for the CharactersScreen I'm interested to zoom the heads of the characters when the customization step is in progress._

_(Ideally the zoom could work with the scroll wheel)_

**Comment by MasterJdR** [comment]
I'm a bit late... is this awesome work downloadable anywhere ?

**Reply by AvoelDev** [reply]
don't think so :(

**Comment by TheSCREWEDSoftware** [comment]
@vaiocti I assume you're not working on this anymore?

**Reply by iThorgrim** [reply]
Oh yes, I'm ready to continue developing this module :D

**Reply by TheSCREWEDSoftware** [reply]
Tel us something whenever you have an update to it!

**Reply by iThorgrim** [reply]
I'm taking the liberty of replying late, if any developers are motivated to work with me on this project it would be a pleasure :D

**Reply by Veraciel** [reply]
I want to use this awesome one !! any way to try please???

**Reply by codefo-O** [reply]
@iThorgrim Hey I know this was awhile ago but is there a repo with this project?
